clients/web/src/templates/layout/layoutGlobalNav.pug

```diff
@@ -5,4 +5,3 @@
         a.g-nav-link(g-target=navItem.target, g-name=navItem.name, href=`#${navItem.target}`)
           i(class=navItem.icon)
           span= navItem.name
-.g-global-nav-fade
```

Comment:
@zachmullen You apparently added this `div` in https://github.com/girder/girder/commit/84604f16485e6032a19f14656dc3428707383bf5#diff-1b1a121412de63cbd86dd297a34998d5 . Is it fine to remove?

Reply:
Interesting. Must have been something we did that added it back. That was the first thing I removed. Yes you can remove.

Reply:
@jtomeck This PR does indeed remove the `.g-global-nav-fade`. Since it was originally added by @zachmullen , I'm now asking whether there's still a reason why he thinks it should remain.

Reply:
The element is fine to be removed, it was a hack for styling because I am not as good at CSS as @jtomeck  :)

I think I copied the implementation from someplace else, might even have been github from many years ago.